### PR TITLE
Update meeting-policies-participants-and-guests.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7665.
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7276.

I've tested it and "People in my organization" is the same as "EveryoneInCompanyExcludingGuests" in PS.
"People in my organization and guests" is the same as "EveryoneInCompany" in PS.

I've also tested the default value and it's "People in my organization and guests"/"EveryoneInCompany" no matter where you create it.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-participants-and-guests.md [hunk 1]
```diff
@@ -59,11 +59,11 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
-|**Everyone in my organization**    |Authenticated users from within the organization, including guest users, join the meeting directly without waiting in the lobby.  Users from trusted organizations and anonymous users wait in the lobby. This is the default setting.           |
-|**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby.           |
-|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby.           |
+|**People in my organization**    |Authenticated users from within the organization, join the meeting directly without waiting in the lobby.  Users from trusted organizations, guest users, and anonymous users wait in the lobby.          |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:64]:
```suggestion
|**People in my organization**    |Authenticated users from within the organization join the meeting directly without waiting in the lobby.  Users from trusted organizations, guest users, and anonymous users wait in the lobby.          |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**